### PR TITLE
First version of SaveBlock3 in debug menu

### PR DESCRIPTION
**Pull request by pkmnsnfrn**
I'm fairly certain this isn't the _right solution, but I figured it's a start. Need to figure out what `maxSb3Size` should be.

### CODE REVIEW

File: include/save.h [hunk 1]
```diff
@@ -17,6 +17,8 @@
 #define SECTOR_ID_SAVEBLOCK2          0
 #define SECTOR_ID_SAVEBLOCK1_START    1
 #define SECTOR_ID_SAVEBLOCK1_END      4
+#define SECTOR_ID_SAVEBLOCK2_START    14
+#define SECTOR_ID_SAVEBLOCK2_END      27
```
Comment (mrgriffin) [inline include/save.h]: I think this has slipped in from a backup-save removal commit?

File: src/debug.c [hunk 2]
```diff
@@ -2071,17 +2071,27 @@ void CheckSaveBlock1Size(struct ScriptContext *ctx)
...
+    u32 maxSb2Size = SECTOR_DATA_SIZE * (SECTOR_ID_SAVEBLOCK2_END - SECTOR_ID_SAVEBLOCK2_START + 1);
     ConvertIntToDecimalStringN(gStringVar1, currSb2Size, STR_CONV_MODE_LEFT_ALIGN, 6);
     ConvertIntToDecimalStringN(gStringVar2, maxSb2Size, STR_CONV_MODE_LEFT_ALIGN, 6);
     ConvertIntToDecimalStringN(gStringVar3, maxSb2Size - currSb2Size, STR_CONV_MODE_LEFT_ALIGN, 6);
 }
 
+void CheckSaveBlock3Size(struct ScriptContext *ctx)
+{
+    u32 currSb3Size = (sizeof(struct SaveBlock3));
+    u32 maxSb3Size = SECTOR_DATA_SIZE;
```
Comment (mrgriffin) [inline src/debug.c]:
```suggestion
    u32 maxSb3Size = SAVE_BLOCK_3_CHUNK_SIZE * NUM_SECTORS_PER_SLOT;
```

**Review by mrgriffin — CHANGES_REQUESTED** [review]
Thanks psf! I've not tested this yet, but have a few changes based on skimming it :)

File: include/save.h (hunk 1 above)
Comment (pkmnsnfrn) [reply include/save.h]: No, this is me trying to guess what sectors these end in for use in `CheckSaveBlock2Size`.

Comment (mrgriffin) [reply include/save.h]: Ah! I think `CheckSaveBlock2Size` was fine, there's just a single sector dedicated to `SaveBlock2` (it's sector 0) :)

**Review by mrgriffin — APPROVED** [review]

**Pull request merged**